### PR TITLE
Update ManagedFusionOffering template and remove storageclassclaim CRD workaround

### PR DESCRIPTION
**Pull request by jilju**
Update ManagedFusionOffering template and remove storageclassclaim CRD workaround. These are the changes to suppport latest builds of agent.

### CODE REVIEW

File: ocs_ci/templates/fusion-aas/managedfusionoffering.yaml.j2 [hunk 1]
```diff
@@ -6,6 +6,6 @@ metadata:
 spec:
   kind: DF
   release: "{{ ocp_version }}"
-  config: 
+  config: |
     usableCapacityInTiB: "{{ size }}"
```
Comment (rewantsoni) [inline ocs_ci/templates/fusion-aas/managedfusionoffering.yaml.j2:10]:
```suggestion
    usableCapacityInTiB: {{ size }}
```
We no longer require "" for size param Sample for [DF](https://github.com/red-hat-storage/managed-fusion-agent/blob/main/config/samples/misf_v1alpha1_managedfusionoffering.yaml)

**Review by rewantsoni — CHANGES_REQUESTED** [review]

File: ocs_ci/templates/fusion-aas/managedfusionoffering.yaml.j2 (hunk 1 above)
Comment (jilju) [reply ocs_ci/templates/fusion-aas/managedfusionoffering.yaml.j2]: Removed the quotes.

**Review by fbalak — DISMISSED** [review]

**Comment by jilju** [comment]
Verified:
Provider: https://ocs4-jenkins-csb-odf-qe.apps.ocp-c1.prod.psi.redhat.com/job/qe-odf-multicluster/1740
Consumer: https://ocs4-jenkins-csb-odf-qe.apps.ocp-c1.prod.psi.redhat.com/job/qe-deploy-ocs-cluster/25373

**Review by fbalak — APPROVED** [review]

**Review by ebondare — APPROVED** [review]

**Review by rewantsoni — APPROVED** [review]

**Pull request merged**